src/itemObjGen.txt

```diff
@@ -8411,4 +8489,4 @@
       count n = 20
     ingredients[]
       name = Aloe
-      count n = 20
\ No newline at end of file
+      count n = 20
```

Comment:
I don't know why this changed. I used the github editor. Maybe it's the EOF changing new lines

Reply:
Hmm... yes that's possible.
As long as the data remains the same shouldn't matter tho.

I'm on Windows but have git set to automatically commit line endings unix style.